css/admin.css

```diff
@@ -0,0 +1,3 @@
+#adminmenu ul > li > a[href="edit.php?post_type=block_lab&page=block-lab-pro"] {
```

Comment:
It's nice how you organized the stying in `css/`. Sometimes I'll output inline `<style>` tags in the markup, but it's cleaner how they're organized here in `css/`.